pitch.md

```diff
@@ -1,44 +1,80 @@
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+I think that similar to SQL, this task is better suited as a query system. Although, the most intuitive reservation system may be a visual, a query system can be the internal DSL for a future GUI project. For calendar users, it will provide a smoother experience than the one currently found on EMS.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+When I try to organize events, it is fairly frustrating to check multiple dates and times or try to do multiple reservations. When I was trying to reserve a MuddVan, I came up with the idea to try something similar and it also aligned with my proposal in free-1.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain? Be careful
 to distinguish between, e.g,. a graphical interface and a linguistic interface._ 
 
+The user would be able to do three basic tasks.
+* Create spaces that can be reserved
+* Reserve spaces for periods of times
+* Query for space availability
+
+The user interacts through text queries. The programming will look similar to other query languages like SQL. 
+
+* Create spaces
+
+This would ideally return a confirmation or an error.
+
+```
+CREATE ROOM "Blue Room" IN "Platt"
+```
+
+* Reserve spaces
+
+This would ideally return either a confirmation or an error.
+
+```
+RESERVE ROOM "Blue Room" FROM "1600" TO "2030" ON "11/28/2015"
+```
+
+* Query for availability
+
+This would ideally return a list of reservations on that day, or a list of available times.
+
+```
+CHECK ROOMS IN "Platt" ON "11/28/2015"
+```
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
 might they be communicated to the user?_
 
+When a query is made, the program parses the query and then returns the appropriate data. It may use a database service like Parse to handle holding information. The data can also be stored locally.
+
+Some errors include querying for names that don't appear. Ideally, the query would return some suggestions but it can also just respond with an error. Another error is that multiple users may attempt to book at the same time, in this case the program should choose one user to give precedence to.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be very easy to create new spaces, reserve spaces, and check the schedule. It should be possible but difficult to write a series of queries that checks which rooms are empty and then books the rooms at the desired time. (On second thought this might be an interesting addition to the language). It should be impossible to do programs such as "hello world" or recursion.
```

Comment:
Regarding sequences of queries, one thing you might do is try to get this to interface well with Unix text utilities by
   - Having it interact with command line arguments, standard input and standard output appropriately
   - Having it read input and print output in simple whitespace-delimited patterns.